File: 02-Python/04-bmi-calculator.py
#!/usr/bin/python

your_weight = int(input("What is your weight? (e.g.: 65) \n"))
your_height = float(input("\nWhat is your height? (e.g.: 1.8) \n"))

def bmi_calculator (weight, height):

    bmi = round((weight /  height**2), 1)

    your_bmi = "Your BMI is "
    so_you = ", so you "

    if bmi < 18.5:
        print(your_bmi + str(bmi) + so_you + "are underweight.") 
    
    if (bmi > 18.5 and bmi < 24.9):
        print(your_bmi + str(bmi) + so_you + "have a normal weight.")

    if bmi > 24.9:
        print(your_bmi + str(bmi) + so_you + "are overweight.")


bmi_calculator(your_weight, your_height)

File: 01-JavaScript/04-bmi-calculator.js
const prompt = require("prompt-sync")({ sigint: true });

const yourWeight = prompt("What is your weight? (e.g.: 65) ");
const yourHeight = prompt("What is your height? (e.g.:1.8) ");

function bmiCalculator(weight, height) {

    const givenWeight = parseInt(weight);
    const givenHeight = parseFloat(height);

    const bmi = Math.round(givenWeight / givenHeight ** 2);

    const yourBMI = "Your BMI is ";

    if (bmi < 18.5) {
        console.log(yourBMI + bmi + ", so you are underweight.");
    }
    if (bmi > 18.5 && bmi < 24.9) {
        console.log(yourBMI + bmi + ", so you have a normal weight.");
    }
    if (bmi > 24.9) {
        console.log(yourBMI + bmi + ", so you are overweight.");
    }

}

bmiCalculator(yourWeight, yourHeight);